bat/tests/06-test-format-bump/run.bats

```diff
@@ -1,14 +1,48 @@
 #!/usr/bin/env bats
 
 # shared test functions
-#load ../../lib/mixerlib
+load ../../lib/mixerlib
 
-#setup() {
-#  global_setup
-#}
+setup() {
+  global_setup
+}
 
-# TODO: fill out once auto format bump implemented
-#@test "Create initial mix 10" {
-#}
+# TODO: This test should run 'mixer build upstream-format' instead
+# once the tooling required to run custom test docker images for
+# is ready.
+@test "Perform upstream format bump on last format boundary" {
+  upstreamver=$(get-last-format-boundary)
+  mixer-init-stripped-down $upstreamver 10
+
+  #Create initial version in the old format
+  mixer-build-all > $LOGDIR/build_all.log
+
+  #Update to new upstream in the new format
+  mixer-upstream-update $(($upstreamver + 10)) > $LOGDIR/upstream_update.log
+
+  #Build +20
+  mixer-build-format-bump-new > $LOGDIR/build_formatbump_new.log
+
+  #Reset upstream for +10 build
+  echo $upstreamver > upstreamversion
+
+  #Build +10
+  mixer-build-format-bump-old > $LOGDIR/build_formatbump_old.log
+
+  #restore upstrean
+  mv upstreamversion.bump upstreamversion
+
+  #check if LAST_VER is set to +20
+  test $(cat update/image/LAST_VER) -eq 30
+
+  #check if +20 previous version is set to +0
+  test $(awk -v FS="previous:" 'NF>1{gsub(/\t/, "", $2); print $2}' update/www/30/Manifest.MoM) -eq 10
+
+  #check if +10 previous version is set to +0
+  test $(awk -v FS="previous:" 'NF>1{gsub(/\t/, "", $2); print $2}' update/www/20/Manifest.MoM) -eq 10
+
+  #check if builder.conf has the +20 format
+  test $(awk -v FS="FORMAT" 'NF>1{gsub(/=/, "",$2); gsub(/"/,"",$2); gsub(/ /,"",$2); print $2}' builder.conf) -eq 2
```

Comment:
```test $(sed -n 's/^FORMAT=//p' builder.conf) -eq 2``` would be my suggestion.

Reply:
This doens't work here. It has to work for both config formats:
`FORMAT=2`
`  FORMAT = "2"`
Thats is why I use just FORMAT as FS and use the rest to remove =, " and <space>

Reply:
OK, so ```test $(sed -n s/^FORMAT[ ="]*\([^"]*\)"?/\1/p) -eq 2```

Reply:
You can also remove all whitespace before parsing the line and use the original suggestion.